### PR TITLE
doc: add minutes for meeting 19 Apr 2017

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Review by jasnell — APPROVED** [review]

**Review by Trott — APPROVED** [review]
LGTM. Thanks for taking the minutes, @mhdawson!

**Comment by joshgav** [comment]
Thanks @mhdawson! LGTM (_edit_: oops, modulo a few minor corrections)

File: meetings/2017-04-19.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# Node Foundation CTC Meeting 2017-04-19
+## Links
+
+* **Audio recording**: <https://www.youtube.com/watch?v=S69FsEpLacc>
```
Comment (joshgav) [inline meetings/2017-04-19.md]: This is the previous week's meeting - this one is https://www.youtube.com/watch?v=DemuVw4XRHQ

File: meetings/2017-04-19.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+* **GitHub Issue**: [CTC#98](https://github.com/nodejs/CTC/issues/105)
```
Comment (joshgav) [inline meetings/2017-04-19.md]: Oops, contradiction between text and link.

File: meetings/2017-04-19.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+* [async_hooks PR](https://github.com/nodejs/node/pull/11883)
+
+## Minutes
+
+### nodejs/node
+
+#### src: update --inspect hint text [#11207](https://github.com/nodejs/node/pull/11207)
+
+* with move of release date for Node Version 8, defer this to next week
+  when Josh can frame the conversation.
```
Comment (joshgav) [inline meetings/2017-04-19.md:93]: sounds good :)

**Review by joshgav — CHANGES_REQUESTED** [review]

File: meetings/2017-04-19.md (hunk 1 above)
Comment (mhdawson) [reply meetings/2017-04-19.md]: Fixed

File: meetings/2017-04-19.md (hunk 2 above)
Comment (mhdawson) [reply meetings/2017-04-19.md]: fixed

**Comment by mhdawson** [comment]
@joshgav pushed commit to address comments.

**Review by joshgav — APPROVED** [review]

**Comment by mhdawson** [comment]
Landed as 90dae984ab5582561731ab1c23d8f24e2a782c9a

**Pull request closed without merging**